test/PrivateComposerInstaller/RemoteFilesystemTest.php

```diff
@@ -25,6 +30,10 @@ public function testExtendsComposerRemoteFilesystem()
 
```

Comment:
```suggestion

    protected function setUp(): void
    {
        if (version_compare(PluginInterface::PLUGIN_API_VERSION, '2.0.0', '>=')) {
            $this->markTestSkipped();
        }

        $this->io = $this->createMock(IOInterface::class);
    }
```

@mcaskill With the Blue Plus sign!

Reply:
Normally, yes. But the `setup()` method is in a greyed-out section (indicating its outside of the PR's changes) and GitHub is not displaying the blue plus sign for me to suggest it.